--- fish-1.6/Makefile.in.orig	2005-04-10 17:34:23.610792088 +0200
+++ fish-1.6/Makefile.in	2005-04-10 18:27:54.849609584 +0200
@@ -187,25 +187,25 @@
 	echo "}" >>$@
 #man -- doc_src/builtin_doc/man/man1/`basename $@ .c`.1 | cat -s | ./gen_hdr2 >>$@
 install: all 
-	$(INSTALL) -m 755 -o 0 -g 0 -d $(DESTDIR)/usr/local/bin/ 
+	$(INSTALL) -m 755  -d $(DESTDIR)/usr/bin/ 
 	for i in fish tokenize set_color xsel-0.9.6/xsel mimedb; do\
-		$(INSTALL) -m 755 -o 0 -g 0 $$i $(DESTDIR)/usr/local/bin; \
+		$(INSTALL) -m 755  $$i $(DESTDIR)/usr/bin; \
 	done;
-	$(INSTALL) -m 755 -o 0 -g 0 -d $(DESTDIR)/etc/fish.d 
-	$(INSTALL) -o 0 -g 0 -m 644 init/fish $(DESTDIR)/etc 
-	$(INSTALL) -o 0 -g 0 -m 644 init/fish_alias.fish $(DESTDIR)/etc/fish.d/;
-	$(INSTALL) -o 0 -g 0 -m 644 init/fish_complete.fish $(DESTDIR)/etc/fish.d/;
-	$(INSTALL) -o 0 -g 0 -m 644 init/fish_inputrc $(DESTDIR)/etc/fish_inputrc;
-	$(INSTALL) -o 0 -g 0 -m 755 -d $(DESTDIR)/usr/share/doc/fish
+	$(INSTALL) -m 755  -d $(DESTDIR)/etc/fish.d 
+	$(INSTALL)  -m 644 init/fish $(DESTDIR)/etc 
+	$(INSTALL)  -m 644 init/fish_alias.fish $(DESTDIR)/etc/fish.d/;
+	$(INSTALL)  -m 644 init/fish_complete.fish $(DESTDIR)/etc/fish.d/;
+	$(INSTALL)  -m 644 init/fish_inputrc $(DESTDIR)/etc/fish_inputrc;
+	$(INSTALL)  -m 755 -d $(DESTDIR)/usr/share/doc/fish
 	for i in user_doc/html/*; do \
-		$(INSTALL) -m 644 -o 0 -g 0 $$i $(DESTDIR)/usr/share/doc/fish/; \
+		$(INSTALL) -m 644  $$i $(DESTDIR)/usr/share/doc/fish/; \
 	done
-	$(INSTALL) -o 0 -g 0 -m 755 -d $(DESTDIR)/usr/local/man/man1
+	$(INSTALL)  -m 755 -d $(DESTDIR)/usr/share/man/man1
 	for i in doc_src/fish.1 xsel-0.9.6/xsel.1x doc_src/builtin_doc/man/man1/mimedb.1 doc_src/builtin_doc/man/man1/set_color.1 doc_src/builtin_doc/man/man1/tokenize.1; do \
-		$(INSTALL) -m 644 -o 0 -g 0 $$i $(DESTDIR)/usr/local/man/man1/; \
+		$(INSTALL) -m 644  $$i $(DESTDIR)/usr/share/man/man1/; \
 	done;
 	echo If you want to use fish as the default shell, remember to first
-	echo add the line \'/usr/local/bin/fish\' to the file \'/etc/shells\'.
+	echo add the line \'/usr/bin/fish\' to the file \'/etc/shells\'.
 
 #Local install
 linstall: all 
@@ -226,7 +226,7 @@
 	echo add the line \'$(HOME)/bin/fish\' to the file \'/etc/shells\'.
 
 uninstall:
-	rm -f $(DESTDIR)/usr/local/bin/fish $(DESTDIR)/usr/local/bin/tokenize $(DESTDIR)/usr/local/bin/set_color $(DESTDIR)/usr/local/bin/xsel $(DESTDIR)/usr/local/bin/mimedb
+	rm -f $(DESTDIR)/usr/bin/fish $(DESTDIR)/usr/local/bin/tokenize $(DESTDIR)/usr/local/bin/set_color $(DESTDIR)/usr/local/bin/xsel $(DESTDIR)/usr/local/bin/mimedb
 	rm -f $(DESTDIR)/etc/fish 
 	rm -f $(DESTDIR)/etc/fish_inputrc
 	rm -f $(DESTDIR)/etc/fish.d/fish_alias $(DESTDIR)/etc/fish.d/fish_complete; 
