--- fish-3.3.1/cmake/Install.cmake.orig	2021-07-06 16:45:37.000000000 +0200
+++ fish-3.3.1/cmake/Install.cmake	2021-07-19 20:04:46.496541738 +0200
@@ -14,7 +14,11 @@
 set(sysconfdir ${CMAKE_INSTALL_SYSCONFDIR})
 set(mandir ${CMAKE_INSTALL_MANDIR})
 
-set(rel_datadir ${CMAKE_INSTALL_DATADIR})
+if(IS_ABSOLUTE ${CMAKE_INSTALL_DATADIR})
+  file(RELATIVE_PATH rel_datadir ${CMAKE_INSTALL_PREFIX} ${CMAKE_INSTALL_DATADIR})
+else()
+  set(rel_datadir ${CMAKE_INSTALL_DATADIR})
+endif()
 set(datadir ${CMAKE_INSTALL_FULL_DATADIR})
 
 set(docdir ${CMAKE_INSTALL_DOCDIR})
